src/code/z_debug.c

```diff
@@ -282,14 +288,14 @@ void func_80063D7C(GraphicsContext* gfxCtx) {
     GfxPrint_Open(&printer, gfx);
 
     if ((OREG(0) == 1) || (OREG(0) == 8)) {
-        func_80063828(&printer);
+        DbCamera_DrawScreenText(&printer);
     }
```

Comment:
Would be nice to name/doc `OREG(0)` here, but don't worry about it if you don't want to. Can always do it later

Reply:
it has some meanings as 1 and 8 that I dont fully understand, and is sometimes treated as a flag and sometimes not.. it just kinda a mess and idk what to call it

Reply:
It's only used in two placed: here in `z_debug.c` and in `Camera_PrintSettings` which are used to print the status/settings/mode/func/bgCamIndex. So all it's doing is printing cam info to the screen (except for no-clip but w/e). So maybe just `R_DBG_CAM_PRINT_TEXT`?

Looking at values:
`OREG(0) = 0` -> doesn't print anything
`OREG(0) = 1` -> print both `DbCamera_DrawScreenText` and `Camera_PrintSettings`
`OREG(0) = 8` -> print only `DbCamera_DrawScreenText`
`OREG(0) = 1 | 8` -> print only `Camera_PrintSettings`
(not sure why they didn't just give each bit to each of the two types of printing instead of weird usages of = and bit checks)

Reply:
im not sure just "print text" captures that face that it can switch between printing a few different things well
seems like something that can prob be named with another more OREG/debug cam related pass

Reply:
What do you mean by a few different things?

Yeah, we can name this in a different pass, just curious what you mean by that.

Reply:
the different meanings for 1 and 8 you describe above. it can print different kinds of information on the screen, and I think the name should probably capture somehow

Reply:
phrased another way, it is not a simple switch that you turn on and it will print all of the information. it has deeper meaning and lets you switch between what will get printed to the screen